sources/packages/backend/workflow/test/2022-2023/fulltime-assessment/eligibility/fulltime-assessment-2022-2023-eligibility-BCAG.e2e-spec.ts

```diff
@@ -23,7 +23,7 @@ describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}-eligibility-BCAG
   // Expected and not expected program length options.
   const EXPECTED_PROGRAM_LENGTH = [
     ProgramLengthOptions.WeeksToLessThanYear,
-    ProgramLengthOptions.OneToTwoYears,
+    ProgramLengthOptions.WeeksTo59Weeks,
```

Comment:
Please take a look at the similar items in the same enum. When the values start with a number it is just converted to text. Can you please follow the same for the new ones?
```ts
export enum ProgramLengthOptions {
  WeeksToLessThanYear = "12WeeksToLessThan1Year",
  WeeksTo59Weeks = "52WeeksTo59Weeks",
  WeeksToTwoYears = "60WeeksToLessThan2Years",
  TwoToThreeYears = "2YearsToLessThan3Years",
  ThreeToFourYears = "3YearsToLessThan4Years",
  FourToFiveYears = "4YearsToLessThan5Years",
  FiveOrMoreYears = "5YearsOrMore",
}
```